test/models/dspace_metadata_test.rb

```diff
@@ -10,43 +10,57 @@ def dss_friendly_thesis(thesis)
     thesis.save
   end
 
-  test 'parses thesis data as DSpace DC' do
+  test 'metadata file contains thesis data' do
     t = theses(:one)
     dss_friendly_thesis(t)
-    dc = DspaceMetadata.new(t).instance_variable_get(:@dc)
-    assert_equal 'MyString', dc['dc.title']
-    assert_equal '2017-09', dc['dc.date.issued']
-    assert_equal 'MyText', dc['dc.description.abstract']
+    serialized = DspaceMetadata.new(t).serialize_dss_metadata
+    unserialized = JSON.parse(serialized)
+
+    assert_equal ['metadata'], unserialized.keys
```

Comment:
This test is repeated down in the bottom of the file, and I'm not sure why - I'm not sure it is a huge problem, I've repeated tests as part of separate sequences to confirm the expected, but I'm not sure I follow the distinction between this test and what's on line 330.